Products/ZenUtils/Auth0/Auth0.py

```diff
@@ -337,6 +341,10 @@ def challenge(self, request, response):
         # sessionInfo doesn't persist the auth_expiration property when modified outside of the initial storage,
         # and a local dictionary isn't available to all zopes.
         if sessionInfo:
+            if len(sessionInfo.roles) < 1:
+                request['RESPONSE'].redirect('/?errcode=1', lock=1)
```

Comment:
Can you put a comment here on how it interacts with the zing UI and the meaning of the error code number.